assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -211,6 +211,8 @@ const useStoryApi = (dataAdapter, { editStoryURL, wpApi }) => {
 
       try {
         const { createdBy, pages, version } = template;
+        const { flags } = window.webStoriesDashboardSettings;
+
         const storyPropsToSave = await getStoryPropsToSave({
```

Comment:
I also updated `getStoryPropsToSave` to accept `flags` as a param because we're also calling this method from the Dashboard too.